substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/VM.java

```diff
@@ -33,14 +32,20 @@
 public final class VM {
 
     public final String version;
+    public final String vendor;
+    public final String vendorUrl;
+    public final String runtimeName;
 
     @Platforms(Platform.HOSTED_ONLY.class)
     public VM(String config) {
         String versionStr = System.getProperty("org.graalvm.version");
         VMError.guarantee(versionStr != null);
         versionStr = "GraalVM " + versionStr;
-        versionStr += " Java " + JavaVersionUtil.JAVA_SPEC;
+        versionStr += " Java " + Runtime.version().toString();
         versionStr += " " + config;
         version = versionStr;
+        vendor = System.getProperty("org.graalvm.vendor", "Oracle Corporation");
+        vendorUrl = System.getProperty("org.graalvm.vendorurl", "https://www.graalvm.org/");
+        runtimeName = System.getProperty("java.runtime.name", "OpenJDK Runtime Environment");
```

Comment:
Wouldn't `"Unknown JDK Runtime Environment"` be more appropriate?
If `java.runtime.name` is not found we might run on something exotic. So let's not pretend.

Reply:
Ack.